src/dispatch/service/models.py

```diff
@@ -51,6 +51,7 @@ class Service(TimeStampMixin, Base):
     is_active = Column(Boolean, default=True)
     name = Column(String)
     type = Column(String, default="pagerduty-oncall")
```

Comment:
Should we define the slug for the oncall plugin in the config and use it as default instead of a hardcoded string?

Reply:
Probably, but we can do that in a separate PR.